src/sentry/static/sentry/app/views/organizationDiscover/result/pagination.jsx

```diff
@@ -9,34 +9,72 @@ export default class Pagination extends React.Component {
   static propTypes = {
     getNextPage: PropTypes.func.isRequired,
     getPreviousPage: PropTypes.func.isRequired,
-    previous: PropTypes.string,
-    next: PropTypes.string,
+    baseQuery: PropTypes.shape({
+      query: PropTypes.shape({
+        limit: PropTypes.number.isRequired,
+      }).isRequired,
+      current: PropTypes.string.isRequired,
+      next: PropTypes.string,
+      previous: PropTypes.string,
+      data: PropTypes.shape({
+        data: PropTypes.array.isRequired,
+      }).isRequired,
+    }).isRequired,
   };
 
+  getPageNumber() {
+    const {baseQuery} = this.props;
+    const startRange = parseInt(baseQuery.current.split(':')[1], 10);
+    const endRange = startRange + baseQuery.query.limit;
+    const dataLength = baseQuery.data.data.length;
+
+    if (dataLength) {
+      const from = startRange + 1;
+      const to = dataLength < baseQuery.query.limit ? from + dataLength : endRange;
```

Comment:
Do we need the ternary? In the case where dataLength == baseQuery.query.limit can't we still use the same formula?